cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1500,4 +1500,22 @@
      */
     public static final String UPDATE_ALLOWED = PREFIX + "update.allowed";
   }
+
+  /**
+   * Constants for capability management
+   */
+  public static final class Capability {
+    /**
+     * Interval for scanning config
+     */
+    public static final String DIR_SCAN_INTERVAL_MINUTES = "capability.dir.scan.interval.minutes";
+    /**
+     * Capability config directory path key
+     */
+    public static final String CONFIG_DIR = "capability.config.dir";
+    /**
+     * Timeout for re-applying capabilities
+     */
+    public static final String APPLY_TIMEOUT_MINUTES = "capability.apply.timeout.minutes";
```

Comment:
I don't see this being used? If not, please remove it and also from the cdap-default.xml

Reply:
Removed.